packages/bpk-component-barchart/src/BpkChartMargin-test.js

```diff
@@ -20,13 +20,15 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import { spacingSm } from 'bpk-tokens/tokens/base.es6';
 
+import { remToPx } from './utils';
 import BpkChartMargin from './BpkChartMargin';
 
+const spacing = remToPx(spacingSm);
 const margin = {
-  top: spacingSm,
-  right: spacingSm,
-  bottom: spacingSm,
-  left: spacingSm,
+  top: spacing,
```

Comment:
I think we are better of changing the `propTypesS` for `BpkCharMargin` from

```javascript

BpkChartMargin.propTypes = {
  children: PropTypes.node.isRequired,
  margin: PropTypes.shape({
    top: PropTypes.number,
    bottom: PropTypes.number,
    left: PropTypes.number,
    right: PropTypes.number,
  }).isRequired,
};
```

to 

```javascript
BpkChartMargin.propTypes = {
  children: PropTypes.node.isRequired,
  margin: PropTypes.shape({
    top: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
    bottom: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
    left: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
    right: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  }).isRequired,
};
```

Reply:
Makes sense, changed it 👍